install/resources/settings.xml

```diff
@@ -1833,10 +1833,18 @@ activity=Order By Last Activity (DESC)]]></optionscode>
 			<settingvalue><![CDATA[username]]></settingvalue>
 			<isdefault>1</isdefault>
 		</setting>
+		<setting name="wolusersperpage">
+			<title>Users Per Page</title>
+			<description><![CDATA[The number of users displayed per page. Set to 0 to use the default.]]></description>
+			<disporder>4</disporder>
+			<optionscode><![CDATA[numeric
+min=0]]></optionscode>
+			<settingvalue><![CDATA[20]]></settingvalue>
+			<isdefault>1</isdefault>
```

Comment:
You forgot the closing `</setting>`

Reply:
Good spot, it was because resolving merge conflict. I will fix this, thx